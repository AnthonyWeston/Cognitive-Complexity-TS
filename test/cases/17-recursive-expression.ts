const i = () => {
    i();
};

const j = () => {
    j();
    j();
};

const k = () => {
    i();
};

const l = function() {
    l();
};

const m = function() {
    m();
    m();
};

const n = function() {
    i();
};

() => {
    i();
};

const o = function p() {
    i();
    o();
    p();
};
