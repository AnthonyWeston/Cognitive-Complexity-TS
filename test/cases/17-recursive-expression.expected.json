{
    "17-recursive-expression.ts": {
        "score": 6,
        "inner": [
            {
                "name": "i",
                "score": 1,
                "line": 1,
                "column": 1
            },
            {
                "name": "j",
                "score": 2,
                "line": 5,
                "column": 1
            },
            {
                "name": "k",
                "score": 0,
                "line": 10,
                "column": 1
            },
            {
                "name": "l",
                "score": 1,
                "line": 14,
                "column": 1
            },
            {
                "name": "m",
                "score": 2,
                "line": 18,
                "column": 1
            },
            {
                "name": "n",
                "score": 0,
                "line": 23,
                "column": 1
            }
        ]
    }
}
